### DISCUSSION TITLE
Is this against the rules?

### DISCUSSION THREAD

**Discussion by csirikk** [the-marenga/sf-scrapbook-helper#39] [Q&A] [answered]
I'm wondering if you can get banned for using this tool

**Comment by the-marenga** [comment, marked as answer]
If you fill your entire scrapbook from 0 to 100% in 15 seconds by repeatedly clicking the best player non stop, then there is a chance a playagames mod/dev might pick up on that and ban you, yes. However, I have made sure the chance of that happending is otherwise as low as possible. The crawling of other players is done with a secondary "throwaway" account, that gets registered, when you log in your player, so **your character never does anything suspicious itself**. Even the crawling accounts, that I use to crawl the initial HoFs of every server, have never been banned, even though they send millions of requests to the server and I have never heard anyone mentioning their account getting banned with this tool. 

If you want to be extra save though, you can just login once, use the "copy optimal battle order" button, to have all the players you need to battle to fill your scrapbook in your clipboard and exit the tool. You can then use the official client and battle the players manually at your own pace.

**Reply by csirikk** [reply]
thank you for your reply
